src/containers/b2b/RequisitionPageMain.tsx

```diff
@@ -639,7 +639,6 @@ class RequisitionPageMain extends Component<RouteComponentProps<RequisitionPageM
                             item={product}
                             hideAvailabilityLabel
                             isTableView
-                            onRemove={() => this.loadRequisitionListData(true)}
```

Comment:
Where is the new Req List data being refreshed then upon remove?

Reply:
@shaunmaharaj 
in cart.lineItem component on remove item is called `handleRemoveBtnClicked()` function where called handleQuantityChange() that is the same onRemove(). So the onRemove() isn't neccesery prop for Req list